### ISSUE TITLE
How to upscale an image?

### ISSUE THREAD

**Issue by SoftologyPro** [philz1337x/clarity-upscaler#3]
I installed with running these commands in an empty directory...

```
git clone https://github.com/philz1337x/clarity-upscaler
cd clairty-upscaler
webui.bat
```

Then launch the UI.  It seems like a clone of Stable Diffusion Web UI (aka auto11111)?

How exactly do I use clarity to upscale and image once the UI opens?

![image](https://github.com/philz1337x/clarity-upscaler/assets/47686889/87f7faa1-0616-4676-909b-ff636e457eda)

**Comment by SoftologyPro** [comment]
Or, even easier, is there a stand alone python script to handle the upscale?
`python upscale.py --input input.png --output ouptut.png --scale 4`

**Comment by lucellent** [comment]
Same thing here. No instructions at all.

**Comment by philz1337x** [comment]
I added some infos in the readme.

**Comment by gustavopch** [comment]
@philz1337x I believe the current info in the README assumes the reader is already familiar with A1111. It would be great if you could ELI5 instead.
